snuba/datasets/entities/metrics.py

```diff
@@ -0,0 +1,34 @@
+from snuba.datasets.plans.single_storage import SingleStorageQueryPlanBuilder
+from typing import Mapping, Sequence
+
+from snuba.clickhouse.columns import ColumnSet
+from snuba.datasets.entity import Entity
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_writable_storage
+from snuba.pipeline.simple_pipeline import SimplePipelineBuilder
+from snuba.query.extensions import QueryExtension
+from snuba.query.processors import QueryProcessor
+
+
+class MetricsEntity(Entity):
```

Comment:
Does not do much but we need a dataset and an entity in order for bootstrap to create the topic .. . which does not seem right

Reply:
Since we will have an enum of topics hopefully we can do away with this soon.

Reply:
Are you saying you don't want people to be able to query this storage though? We will need a dataset entity for that.

Reply:
> Are you saying you don't want people to be able to query this storage though?

No, I mean that I did not create a translator yet. I wanted to start with the storages only in this PR to keep it small but the entity is needed for the bootstrap.

Reply:
It's no longer required for bootstrap at least, up to you whether you want to keep this entity here.

Reply:
Right, well at this point I would leave it here as I will be using it in the next PR anyway.